packages/mds-transactions-service/@types/index.ts

```diff
@@ -103,10 +109,23 @@ export const TRANSACTION_STATUS_TYPES = [
 
 export type TRANSACTION_STATUS_TYPE = typeof TRANSACTION_STATUS_TYPES[number]
 
+export const SORTABLE_COLUMNS = ['timestamp'] as const
+export type SORTABLE_COLUMN = typeof SORTABLE_COLUMNS[number]
+
+export const SORT_DIRECTIONS = ['ASC', 'DESC'] as const
+export type SORT_DIRECTION = typeof SORT_DIRECTIONS[number]
```

Comment:
If you don't prefer the plural/singular thing, you can have a type/value of the same name and TS figures out which you need based on context, so you can have:
```ts
export const SORTABLE_COLUMN = ['timestamp'] as const
export type SORTABLE_COLUMN = typeof SORTABLE_COLUMN[number]
```
Also, you can use this format for the const assertion
```ts
export const SORT_DIRECTIONS = <const>['ASC', 'DESC']
```

Reply:
Ooh, that's news to me! Very fancy, will try it out